src/test/java/pageComponents/menu/OrderConfirmationPageComponent.java

```diff
@@ -70,11 +73,39 @@ public String getDate() {
         return datePickerField.getAttribute("value");
     }
 
-    public void cancelOrder()    {
+    public void cancelOrder() {
```

Comment:
do you really need this method if you already have another one?

Reply:
We needed to overload this method because only I need a waiter for this method.